app/controllers/HasCommodityCodeChangeController.scala

```diff
@@ -55,9 +55,14 @@ class HasCommodityCodeChangeController @Inject() (
   def onPageLoad(mode: Mode, recordId: String): Action[AnyContent] =
     (identify andThen profileAuth andThen getData andThen requireData).async { implicit request =>
```

Comment:
not here but HasGoodsDescriptionChangeController - move audit changes from onSubmit to onPageLoad